Source/WebCore/Modules/WebGPU/GPUTexture.cpp

```diff
@@ -102,7 +102,7 @@ ExceptionOr<Ref<GPUTextureView>> GPUTexture::createView(const std::optional<GPUT
 {
     if (textureViewDescriptor.has_value() && textureViewDescriptor->format.has_value()) {
         if (!m_device->isSupportedFormat(*textureViewDescriptor->format))
-            return Exception { ExceptionCode::TypeError, "Unsupported texture format."_s };
+            return Exception { ExceptionCode::TypeError, "GPUTexture.createView: Unsupported texture format."_s };
```

Comment:
The non-obvious logging was discovered with the repro for this bug, it is not strictly necessary to make this change and the one below it, but probably not worth creating a PR simply for these two string changes